js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +282,30 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        shipsFromCountry: app.settings.get('country'),
       }));
 
       super.render();
 
       this._$deleteListing = null;
       this._$shipsFreeBanner = null;
       this._$popInMessages = null;
+      this._$photoSection = null;
```

Comment:
A local pickup setting option looks a little bare-bone to me. I think we need some other text / indicator to make it clearer what's going on:

![image](https://cloud.githubusercontent.com/assets/794517/21122705/b8a21f5e-c088-11e6-8500-b0b080ef34c5.png)

paging senior lobo @morebrownies